circuit_maintenance_parser/parsers/sparkle.py

```diff
@@ -0,0 +1,94 @@
+"""Sparkle parser."""
+import logging
+from dateutil import parser
+
+from circuit_maintenance_parser.errors import ParserError
+from circuit_maintenance_parser.parser import CircuitImpact, Html, Impact, Status
+
+logger = logging.getLogger(__name__)
+
+
+class HtmlParserSparkle1(Html):
+    """Notifications HTML Parser 1 for Sparkle notifications.
+
+    Example:
+        <table>
+            <tbody>
+                <tr>
+                    <td><p></p>Maintenance ID</td>
+                    <td><p></p>1111 / 2222</td>
+                </tr>
+                <tr>
+                    <td><p></p>Start Date/Time (UTC) Day 1</td>
+                    <td><p></p>08/10/2021 03:00 UTC</td>
+                </tr>
+                <tr>
+                    <td><p></p>End Date/Time (UTC) Day 1</td>
+                    <td><p></p>08/10/2021 11:00 UTC</td>
+                </tr>
+                <tr>
+                    <td><p></p>Start Date/Time (UTC) Day 2</td>
+                    <td><p></p>08/11/2021 03:00 UTC</td>
+                </tr>
+                <tr>
+                    <td><p></p>End Date/Time (UTC) Day 2</td>
+                    <td><p></p>08/11/2021 11:00 UTC</td>
+                </tr>
+                ...
+            </tbody>
+        </table>
+    """
+
+    def parse_html(self, soup):
+        """Execute parsing."""
+        data = {}
+        try:
+            return self.parse_tables(soup.find_all("table"), data)
+        except Exception as exc:
+            raise ParserError from exc
+
+    def clean_string(self, string):
+        """Remove hex characters and new lines."""
```

Comment:
For my own education, can you point out in the test files some examples of hex characters that are needing to be removed? I'm not sure I understand what is meant by that.

Reply:
Inside the sparkle test you can see strange charecters being used. Maybe they arent HEX, i can change the name if needed.
`1111111=C2=A0/ 22222=C2=A0/ 33333` is in the sparkle test where its really meant to be `1111111 / 22222 / 33333`

Reply:
Hm. I thought those were supposed to be removed by our use of `quopri` in the base HTML parser class. Are you seeing cases where these characters are still present in the processed HTML being passed through to the Sparkle parser?

Reply:
Yeah, I think I saw this in another parser but unsure which as it was a while ago.